public/content/lessons/2016/span/index.mdx

```diff
@@ -7,120 +7,151 @@ video: k7RM-ot2NWY
+
+<Figure
+  image="./figures/introduction/CoordinatesAsScalarsOfBasisVectors.svg"
+/>
+
+There's also a more technical definition of basis, but we'll get to that later. Framing our familiar coordinate system in terms of these two special basis vectors raises an interesting and subtle point: We could choose a different pair of basis vectors and get a perfectly reasonable *new* coordinate system.
 
 ## Choosing Different Basis Vectors
 
 For example, take some vector pointing up and to the right, along with a vector pointing down and to the right. 
 
 <Figure
-  image="alternate_basis.svg"
+  image="./figures/different-basis-vectors/DifferentBasisVectors.svg"
+/>
+
+Take a moment to think about all the different vectors you can get by choosing two scalars, using each to scale one of the vectors, then adding them. Which two-dimensional vectors can you reach by altering your choice of scalars?
+
+<Figure
+  image="./figures/different-basis-vectors/DifferentBasisSpanExample1.svg" 
+/>
+
+<Figure
+  image="./figures/different-basis-vectors/DifferentBasisSpanExample2.svg" 
+/>
+
+The answer is, for these two vectors, you can describe *every* possible two-dimensional vector this way, and it's a good puzzle to contemplate why. A new pair of basis vectors like this still gives you a way to go back and forth between pairs of numbers and two-dimensional vectors, but the association is definitely different from the version you get using the standard basis of $\hat{\imath}$ and $\hat{\jmath}$.
+
+<Figure
+  image="./figures/different-basis-vectors/DifferentBasisSpanExample3.svg" 
+  caption="$\left[\begin{array}{c}-0.8 \\ 1.3\end{array}\right] \neq\left[\begin{array}{c}3.1 \\ -2.9\end{array}\right]$"
 />
 
-In the above figure, $\overrightarrow{\mathbf{v}}=\begin{bmatrix}1\\ 2\end{bmatrix}$ and $\overrightarrow{\mathbf{w}}=\begin{bmatrix}3\\ -1\end{bmatrix}$.
+We'll go into much more detail on this point in a <LessonLink id="change-of-basis">later chapter</LessonLink>, describing the relationship between different coordinate systems, but for now we just want you to appreciate that any way to describe vectors numerically depends on your choice of basis vectors.
 
 <Question
-  question="What values of the scalars $\alpha$ and $\beta$ satisfy the following equation? $\alpha\vec{\mathbf{v}}+\beta\vec{\mathbf{w}}=5\hat i -\frac12\hat j$"
+  question="Given the equation $\alpha \vec{\mathbf{v}} + \beta \vec{\mathbf{w}} =5\hat{\imath} -\frac12\hat{\jmath}$, where $\vec{\mathbf{v}} = \left[\begin{array}{c} 1 \\ 2 \end{array}\right]$ and $\vec{\mathbf{w}} = \left[\begin{array}{c} 3 \\ -1 \end{array}\right]$, what are the values of the scalars $\alpha$ and $\beta$?"
```

Comment:
I think there is a general question of when is the appropriate time to introduce solving a linear system of equations to the student. This is a question from River, see the new question at the end of this lesson as well as the question "formal properties" section in the revised "linear transformations" lesson. I think the answer may be at the end of this article, but maybe we should preface or introduce the idea.

Reply:
I removed this earlier question in favor of the later one.